terraform/stacks/main/variables.tf

```diff
@@ -33,25 +33,27 @@ variable "account_id" {}
 variable "remote_state_bucket" {}
 
 variable "target_stack_name" {
-    default = "tooling"
+  default = "tooling"
 }
 
-variable "main_cert_name" {
-    default="star-fr-cloud-gov-06-16"
-}
+variable "main_cert_name" {}
 
-variable "apps_cert_name" {
-    default="star-app-cloud-gov-16"
-}
+variable "apps_cert_name" {}
 
 variable "services_cidr_1" {}
 variable "services_cidr_2" {}
 variable "kubernetes_cluster_id" {}
 
 variable "restricted_ingress_web_cidrs" {}
 
-/* Variables for Diego Deployment */
+variable "client_elb_count" {
+  default = 0
+}
+variable "client_elb_cert_name" {
```

Comment:
Any reason not to just name this exactly what it is? 18f_gov_elb? instead of client_elb?